FirebaseStorage/Tests/Unit/FIRStorageListTests.m

```diff
@@ -152,6 +152,42 @@ - (void)testListWithPageSizeAndPageToken {
   [FIRStorageTestHelpers waitForExpectation:self];
 }
 
+- (void)testPercentEncodesPlusToken {
+  XCTestExpectation *expectation = [self expectationWithDescription:@"testPercentEncodesPlusToken"];
+  NSURL *expectedURL = [NSURL URLWithString:@"https://firebasestorage.googleapis.com/v0/b/bucket/"
+                                            @"o?prefix=%2Bfoo/&delimiter=/"];
+
+  self.fetcherService.testBlock =
+      ^(GTMSessionFetcher *fetcher, GTMSessionFetcherTestResponse response) {
```

Comment:
Can you avoid the pragma with an approach like #6693?

Reply:
I am getting an unused variable warning, and I don't see how the code you linked to doesn't generate the same warning. Is there any trick I need to employ?